apps/search/style/search.css

```diff
@@ -61,14 +56,16 @@ body, a {
 #suggestions-provider {
   color: #00aac5;
   position: relative;
-  padding: 0 2rem 0 1rem;
+  padding: 0;
+  -moz-padding-end: 2rem;
+  -moz-padding-start: 1rem;
 }
 
 #suggestions-provider::before {
   content: ' ';
-  position: absolute;
-  bottom: 0.2rem;
-  right: .8rem;
+  position: relative;
```

Comment:
This isn't really working. The icon needs to be displayed on the right in LTR mode, and the left in RTL. Relative positioning makes this hard, and right now it always displays on the left. One alternative is to use absolute positioning and use `offset-inline-end` (which is what Ahmed did). But I'll leave the final fix up to you.